lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -155,7 +155,7 @@ private NettyMessagingTransport(
         .option(ChannelOption.SO_REUSEADDR, true)
         .childOption(ChannelOption.SO_KEEPALIVE, true);
 
-    LOG.log(Level.FINE, "Binding to {0}:{1}", new Object[] {host, listenPort});
```

Comment:
>FINE [](start = 18, length = 4)

Change back to FINEST before sending pull request to apache/master. I left it here as it is helps me in debugging the containers feature.